amazon_cloud_code_generator/data/plugins/module_utils/utils.py

```diff
@@ -1,15 +1,26 @@
 import re
 import json
 import functools
+import traceback
 from typing import Iterable, List, Dict, Union
 
+JSON_PATCH_IMPORT_ERR = None
+try:
+    import jsonpatch
```

Comment:
We need to make sure we add this to the module requirements on the amazon.cloud repo. Could you either create an issue over there, or create a PR that's dependent on this one?

Reply:
I opened a PR https://github.com/ansible-collections/amazon.cloud/pull/22